helpers/deproxy.py

```diff
@@ -380,7 +390,6 @@ def create(first_line, headers, date=None, srv_version=None, body=None):
             end = ['', body]
         return '\r\n'.join([first_line] + headers + end)
 
-
```

Comment:
Python has no braces, so we use double empty lines between classes according to https://www.python.org/dev/peps/pep-0008/#blank-lines